src/App.tsx

```diff
@@ -55,6 +55,7 @@ function App() {
   }
 
   const runContractCompilation = async (currentFileContent: string) => {
```

Comment:
Shouldn't we add await before runContractCompilation(currentFileContent); at line 54:

Reply:
I think no because we don't have any dependent action after calling runContractCompilation(). So we don't need to block main execution thread.